src/AppAgent.ts

```diff
@@ -91,27 +88,27 @@ class AppAgent {
                         global.txn = new Transaction({
                             version: process.env.AWS_LAMBDA_FUNCTION_VERSION as string,
                             appKey: appkey || '',
-                            transactionName: requestID,
-                            transactionType: process.env.AWS_LAMBDA_FUNCTION_NAME as string,
-                            uniqueClientId: uuid
+                            transactionName: process.env.AWS_LAMBDA_FUNCTION_NAME as string,
+                            transactionType: 'Lambda',
+                            uniqueClientId: requestID
                         }, findEventData.beaconProperties);
 
                     } else if (findHeader.headersFound) {
                         global.txn = new Transaction({
                             version: process.env.AWS_LAMBDA_FUNCTION_VERSION as string,
                             appKey: appkey || '',
-                            transactionName: requestID,
-                            transactionType: process.env.AWS_LAMBDA_FUNCTION_NAME as string,
-                            uniqueClientId: uuid
+                            transactionName: process.env.AWS_LAMBDA_FUNCTION_NAME as string,
+                            transactionType: 'Lambda',
+                            uniqueClientId: requestID
                         }, findHeader.beaconProperties);
 
                     } else {
                         global.txn = new Transaction({
                             version: process.env.AWS_LAMBDA_FUNCTION_VERSION as string,
                             appKey: appkey || '',
-                            transactionName: requestID,
-                            transactionType: process.env.AWS_LAMBDA_FUNCTION_NAME as string,
-                            uniqueClientId: uuid
+                            transactionName: process.env.AWS_LAMBDA_FUNCTION_NAME as string,
+                            transactionType: 'Lambda',
+                            uniqueClientId: requestID
```

Comment:
this could could be refactored to reduce duplications, it seems the only thing that changes is the second parameter i.e. findHeader.beaconProperties